### PR TITLE
client: Allow ctrl+click to open a new tab

### LINKED ISSUE #104: Ctrl Clicking or Middle Clicking links does not open them in a new tab
For example holding control and clicking Sketch.sh in the top left replaces the current page with the homepage rather than opening the homepage in a new tab. This happens on all other links on the site I can find too.

This happens for me in Firefox, not sure about other browsers.

### PR DESCRIPTION
**Pull request by thangngoc89**
This fixes #104

### CODE REVIEW

File: client/src/Router.re [hunk 1]
```diff
@@ -108,10 +108,12 @@ module LinkUnsafe = {
         ?title
         href
         onClick=(
-          self.handle((event, _self) => {
-            event->ReactEvent.Mouse.preventDefault;
-            pushUnsafe(href);
-          })
+          self.handle((event, _self) =>
+            if (!event->ReactEvent.Mouse.ctrlKey) {
```
Comment (kyldvs) [inline client/src/Router.re]: You probably also want a condition for middle click on the mouse

Comment (thangngoc89) [reply client/src/Router.re]: @kyldvs Thank you for reminding me.

**Comment by thangngoc89** [comment]
@kyldvs could you test with https://rtop-alsoqmiblv.now.sh/ (both middle click and ctrl+click) 

Because I believe click event doesn't handle middle click

**Comment by kyldvs** [comment]
ctrl+click works there, but not middle click

**Comment by thangngoc89** [comment]
@kyldvs Alright. Different behavior between Chrome and Firefox. I can reproduce it as well

**Comment by thangngoc89** [comment]
@kyldvs could you give https://rtop-groeuajxfe.now.sh/ another try ?

**Comment by kyldvs** [comment]
Awesome, both middle click and ctrl click work now.

**Comment by thangngoc89** [comment]
@kyldvs Awesome. It should be live in a few minutes

**Pull request merged**